src/main/java/frc/robot/drive/DriveModes.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.drive;
+
+public class DriveModes {
+    public static DrivePower arcadedrive(double joystickY, double steerAngle, 
+    double currentLeftPower, double currentRightPower, boolean isDrivingForward) {
+
+        double adjustedInput = Functions.calcForwardPower(joystickY);
+        double adjustedSteering = Functions.calcSteeringPower(steerAngle);
+
+        if (joystickY < 0 && adjustedInput > 0) {
+            adjustedInput *= -1;
+        }
+
+        double targetLeftPower = 0;
+        double targetRightPower = 0;
+
+        if (steerAngle > 0) {
```

Comment:
Again, we took the absolute value of the sinusoid when calculating adjustedSteering, and now we're adding the sign back in.